data/serializers.py

```diff
@@ -87,6 +88,19 @@ class Meta:
         fields = ('id', 'username', 'first_name', 'last_name', 'email',)
 
 
+class TokenSerializer(serializers.ModelSerializer):
+    id = serializers.SerializerMethodField("get_key")
```

Comment:
FYI if you don't need an actual method to run, you can use a simple `CharField` with the `source` arg:

```
id = serializers.CharField(source='key', read_only=True)
```